docs/appendices/release-notes/unreleased.rst

```diff
@@ -100,6 +100,10 @@ Changes
 Fixes
 =====
 
+- Fixed an issue that prevented aggregations or grouping operations on virtual
+  tables to run parallel on shard level, even if the inner query would support
+  it.
```

Comment:
Can we phrase this more into user facing direction? Something like "... that resulted in poor performance when ..."?
Have no perfect suggestion but maybe you could come up with one.

Reply:
Sorry, just saw this. I'll follow up on it after the other PRs are all merged.